tests/integration/questionnaire/test_questionnaire_sign_out.py

```diff
@@ -0,0 +1,47 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestSignOut(IntegrationTestCase):
+    def test_sign_out_not_signed_in_redirects_to_signed_out_page(self):
+        self.get("/sign-out")
+        self.assertInUrl("/signed-out")
+
+    def test_sign_out_without_log_out_url_redirects_to_signed_out_page(self):
+        self.launchSurvey("test_textfield")
+        self.get("/sign-out")
+        self.assertInUrl("/signed-out")
+
+    def test_sign_out_with_log_out_url_redirects_to_log_out_url(self):
+        self.launchSurvey(
+            "test_textfield", account_service_log_out_url="https://localhost/logout"
+        )
+        self.get("/sign-out")
+        self.assertInUrl("/logout")
+
+    def test_sign_out_after_navigating_backwards(self):
```

Comment:
This seems a tests for resuming and not signing out

Reply:
As discussed, I have split out the resume tests from the sign out tests.